db/migrate/20171222120314_add_constraints_on_discovery_rules_hostgroups.rb

```diff
@@ -0,0 +1,14 @@
+ class AddConstraintsOnDiscoveryRulesHostgroups < ActiveRecord::Migration[4.2]
+  
+   def change
+    execute "DELETE from discovery_rules dr where NOT EXISTS (select NULL from hostgroups hg where hg.id = dr.hostgroup_id);"
+
+    add_foreign_key "discovery_rules", "hostgroups", name: "discovery_rules_hostgroup_id_fk", :column => "hostgroup_id"
+
+    # turn on Foreign Key checks in MySQL only
+    if ActiveRecord::Base.connection.adapter_name.downcase.starts_with? 'mysql'
+      ActiveRecord::Migration.execute "SET FOREIGN_KEY_CHECKS=1;"
+    end
```

Comment:
Remove this block please, this does not make any sense. It's a session-based setting.

Reply:
Sorry, It just slipped out of my mind. Will do that.